### DISCUSSION TITLE
OSM Entrances

### DISCUSSION THREAD

**Discussion by JBisc** [OvertureMaps/data#290] [Discussion]
Sorry for using the issue area for feature requests, but I found no better place.

Is it planned to make also the OSM building entrances available in the data?

**Comment by jenningsanderson** [comment]
Thanks for the question @JBisc; we have put _some_ thought into this. Currently we only have `building_part` as an additional type in the `buildings` theme, but there is certainly discussion of extending this further, for example: Defining a proper relationship between an address, a place, a building entrance, and a building via stable IDs (GERS). 

I'll leave this discussion open here for any updates.
